src/components/Toast.jsx

```diff
@@ -0,0 +1,23 @@
+import { Alert, Snackbar } from "@mui/material";
+function Toast({ open, onClose, message, severity }) {
```

Comment:
Delvin has request that we stick to defining components using the "const" keword  ie arrow function , instead of "function" keword. This is to ensure consistency in the code base

Reply:
Fixed